docker/cheqd_node/mainnet_launch/.env

```diff
@@ -0,0 +1,10 @@
ImNsaWVudHMiOiBbXSwKICAgICAgICAiY2xpZW50c19jb25zZW5zdXMiOiBbXSwKICAgICAgICAiY2xpZW50c19tZXRhZGF0YSI6IFtdLAogICAgICAgICJwYXJhbXMiOiB7CiAgICAgICAgICAiYWxsb3dlZF9jbGllbnRzIjogWwogICAgICAgICAgICAiMDYtc29sb21hY2hpbmUiLAogICAgICAgICAgICAiMDctdGVuZGVybWludCIKICAgICAgICAgIF0KICAgICAgICB9LAogICAgICAgICJjcmVhdGVfbG9jYWxob3N0IjogZmFsc2UsCiAgICAgICAgIm5leHRfY2xpZW50X3NlcXVlbmNlIjogIjAiCiAgICAgIH0sCiAgICAgICJjb25uZWN0aW9uX2dlbmVzaXMiOiB7CiAgICAgICAgImNvbm5lY3Rpb25zIjogW10sCiAgICAgICAgImNsaWVudF9jb25uZWN0aW9uX3BhdGhzIjogW10sCiAgICAgICAgIm5leHRfY29ubmVjdGlvbl9zZXF1ZW5jZSI6ICIwIiwKICAgICAgICAicGFyYW1zIjogewogICAgICAgICAgIm1heF9leHBlY3RlZF90aW1lX3Blcl9ibG9jayI6ICIzMDAwMDAwMDAwMCIKICAgICAgICB9CiAgICAgIH0sCiAgICAgICJjaGFubmVsX2dlbmVzaXMiOiB7CiAgICAgICAgImNoYW5uZWxzIjogW10sCiAgICAgICAgImFja25vd2xlZGdlbWVudHMiOiBbXSwKICAgICAgICAiY29tbWl0bWVudHMiOiBbXSwKICAgICAgICAicmVjZWlwdHMiOiBbXSwKICAgICAgICAic2VuZF9zZXF1ZW5jZXMiOiBbXSwKICAgICAgICAicmVjdl9zZXF1ZW5jZXMiOiBbXSwKICAgICAgICAiYWNrX3NlcXVlbmNlcyI6IFtdLAogICAgICAgICJuZXh0X2NoYW5uZWxfc2VxdWVuY2UiOiAiMCIKICAgICAgfQogICAgfSwKICAgICJtaW50IjogewogICAgICAibWludGVyIjogewogICAgICAgICJpbmZsYXRpb24iOiAiMC4wMTAwMDAwMDAwMDAwMDAwMDAiLAogICAgICAgICJhbm51YWxfcHJvdmlzaW9ucyI6ICIwLjAwMDAwMDAwMDAwMDAwMDAwMCIKICAgICAgfSwKICAgICAgInBhcmFtcyI6IHsKICAgICAgICAibWludF9kZW5vbSI6ICJuY2hlcSIsCiAgICAgICAgImluZmxhdGlvbl9yYXRlX2NoYW5nZSI6ICIwLjA0NTAwMDAwMDAwMDAwMDAwMCIsCiAgICAgICAgImluZmxhdGlvbl9tYXgiOiAiMC4wNDAwMDAwMDAwMDAwMDAwMDAiLAogICAgICAgICJpbmZsYXRpb25fbWluIjogIjAuMDEwMDAwMDAwMDAwMDAwMDAwIiwKICAgICAgICAiZ29hbF9ib25kZWQiOiAiMC42MDAwMDAwMDAwMDAwMDAwMDAiLAogICAgICAgICJibG9ja3NfcGVyX3llYXIiOiAiMzE1NTc2MCIKICAgICAgfQogICAgfSwKICAgICJwYXJhbXMiOiBudWxsLAogICAgInNsYXNoaW5nIjogewogICAgICAicGFyYW1zIjogewogICAgICAgICJzaWduZWRfYmxvY2tzX3dpbmRvdyI6ICIyNTkyMCIsCiAgICAgICAgIm1pbl9zaWduZWRfcGVyX3dpbmRvdyI6ICIwLjUwMDAwMDAwMDAwMDAwMDAwMCIsCiAgICAgICAgImRvd250aW1lX2phaWxfZHVyYXRpb24iOiAiNjAwcyIsCiAgICAgICAgInNsYXNoX2ZyYWN0aW9uX2RvdWJsZV9zaWduIjogIjAuMDUwMDAwMDAwMDAwMDAwMDAwIiwKICAgICAgICAic2xhc2hfZnJhY3Rpb25fZG93bnRpbWUiOiAiMC4wMTAwMDAwMDAwMDAwMDAwMDAiCiAgICAgIH0sCiAgICAgICJzaWduaW5nX2luZm9zIjogW10sCiAgICAgICJtaXNzZWRfYmxvY2tzIjogW10KICAgIH0sCiAgICAic3Rha2luZyI6IHsKICAgICAgInBhcmFtcyI6IHsKICAgICAgICAidW5ib25kaW5nX3RpbWUiOiAiMTIxMDAwMHMiLAogICAgICAgICJtYXhfdmFsaWRhdG9ycyI6IDEyNSwKICAgICAgICAibWF4X2VudHJpZXMiOiA3LAogICAgICAgICJoaXN0b3JpY2FsX2VudHJpZXMiOiAxMDAwMCwKICAgICAgICAiYm9uZF9kZW5vbSI6ICJuY2hlcSIKICAgICAgfSwKICAgICAgImxhc3RfdG90YWxfcG93ZXIiOiAiMCIsCiAgICAgICJsYXN0X3ZhbGlkYXRvcl9wb3dlcnMiOiBbXSwKICAgICAgInZhbGlkYXRvcnMiOiBbXSwKICAgICAgImRlbGVnYXRpb25zIjogW10sCiAgICAgICJ1bmJvbmRpbmdfZGVsZWdhdGlvbnMiOiBbXSwKICAgICAgInJlZGVsZWdhdGlvbnMiOiBbXSwKICAgICAgImV4cG9ydGVkIjogZmFsc2UKICAgIH0sCiAgICAidHJhbnNmZXIiOiB7CiAgICAgICJwb3J0X2lkIjogInRyYW5zZmVyIiwKICAgICAgImRlbm9tX3RyYWNlcyI6IFtdLAogICAgICAicGFyYW1zIjogewogICAgICAgICJzZW5kX2VuYWJsZWQiOiB0cnVlLAogICAgICAgICJyZWNlaXZlX2VuYWJsZWQiOiB0cnVlCiAgICAgIH0KICAgIH0sCiAgICAidXBncmFkZSI6IHt9LAogICAgInZlc3RpbmciOiB7fQogIH0KfQo=
```

Comment:
Ah so I wouldn't want to pass it this way. Basically the better way to pass a file is to use a -v or volume argument to the Docker container, and then copying it over to the path needed. This way anyone can download the latest genesis file, seed file etc and pass it to the Docker container as a volume argumment. See this example: https://www.docker.com/blog/how-to-use-the-official-nginx-docker-image/#:~:text=%24%20docker%20run%20%2Dit%20%2D%2Drm%20%2Dd%20%2Dp%208080%3A80%20%2D%2Dname%20web%20%2Dv%20~/site%2Dcontent%3A/usr/share/nginx/html%20nginx

It provides a clean and consistent way of specifying config files that is consistent.